contrib/kind.yaml.j2

```diff
@@ -1,7 +1,5 @@
 kind: Cluster
 apiVersion: kind.x-k8s.io/v1alpha4
-featureGates:
```

Comment:
hmm I am confused a little as to why this was added in first commit and is now deleted in 4th commit?

Reply:
did a bad job of cleaning my commits up, sorry for the confusion

Reply:
fixed

Reply:
> @bpickard22 : qq re https://github.com/ovn-org/ovn-kubernetes/issues/4269
If the feature is enabled by default aren't we supposed to turn off the feature gate? why is this set to true? or does this mean the gate is ON, which means the feature is off?

to answer this, not turning on the feature gate turn the feature on, but the issue is that the k8s e2e for this is running regardless of if the feature is on or not, and we dont currently support this in ovn-k. I opened an issue to track this -> https://github.com/ovn-org/ovn-kubernetes/issues/4269 

i know we had thought that turning on the feature gate would handle the this, but @martinkennelly and i explored this option and decided to skip the test for now because it broke dualstack and live migration as well as caused some issues with kubevirt, i will work on enabling it when i have more time if that is fine with you and @trozet

Reply:
I think this explains it better than I did before